packages/samples/headless-commerce-ssr-remix/app/components/sort.tsx

```diff
@@ -0,0 +1,51 @@
+'use client';
+
+// import {SortBy, SortCriterion} from '@coveo/headless-react/ssr-commerce';
+import {useSort} from '@/lib/commerce-engine';
+import {SortBy, SortCriterion} from '@coveo/headless-react/ssr-commerce';
+
+export default function Sort() {
```

Comment:
Could we add this component on the search page too? it's only on product lists right now.